nmpc_mhe/pyomo_dae/DynGen_pyDAE.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, d_mod, hi_t, states, controls, **kwargs):
         self.ipopt_executable = kwargs.get('ipopt_executable', None)
         self.dot_driver_executable = kwargs.get('dot_driver_executable', None)
         override_solver_check = kwargs.get('override_solver_check', False)
-
```

Comment:
Which `k_aug` version is required?

Reply:
Oh I don't even know there are different versions of `k_aug` releases......
But I started using `k_aug` from April. Did you add anything new after that?
Otherwise, I think I use the latest version of `k_aug`.